lib/dest/make-dirs.js

```diff
@@ -4,13 +4,12 @@ var through = require('through2');
 var valueOrFunction = require('value-or-function');
 
 var fo = require('../file-operations');
-var writeContents = require('./write-contents');
 
 var number = valueOrFunction.number;
 
-function saveFileStream(opt) {
+function makeDirs(opt) {
```

Comment:
This is fine, I can probably start to refactor a mkdirp-stream module out of here.